### ISSUE TITLE
Formatting in logs is demolished due to changes made by GS

### ISSUE THREAD

**Issue by 2Abendsegler** [2Abendsegler/GClh#759]
**Describe the bug**
Die Formatierung in den Logs und vor allem im User Bereich vor den Logs ist auseinandergezogen und demoliert.

**To Reproduce**
1. Go to a Listing with logs

![grafik](https://user-images.githubusercontent.com/22332216/44142052-448d7338-a07f-11e8-959b-ef8198432e13.png)

Zuvor (aus overview_screenshots.md):
![grafik](https://user-images.githubusercontent.com/22332216/44142077-63b9fca4-a07f-11e8-8663-f954cf5fc6f0.png)

**Desktop (please complete the following information):**
 - Windows 8.1
 - FF 
 - 61.0.1 (64-Bit)

**Comment by Ruko2010** [comment]
GS nervt langsam mit den ganzen "kleinen" Änderungen...

**Comment by 2Abendsegler** [comment]
Ja, und manchmal könnte man meinen die haben es auf uns abgesehen.

**Comment by 2Abendsegler** [comment]
So sieht es jetzt aus:
![grafik](https://user-images.githubusercontent.com/22332216/44143405-db4ebdf0-a083-11e8-80a3-f587a2bf188f.png)
